spyder_env_manager/spyder/plugin.py

```diff
@@ -65,8 +67,11 @@ def on_preferences_available(self):
         preferences.register_plugin_preferences(self)
 
     def check_compatibility(self):
-        valid = True
-        message = ""  # Note: Remember to use _("") to localize the string
+        message = _("")
+        conda_like_executable_path = self.get_conf("conda_file_executable_path")
+        valid = conda_like_executable_path and Path(conda_like_executable_path).exists()
+        if not valid:
+            message = _("Unable to find conda-like executable!")
```

Comment:
I though the plugin was intended to use in an initial phase just the conda-like backend since the initial idea was to make it use the micromamba executable that comes bundled with the installers or maybe I misunderstood the idea?

Reply:
That's fine. I thought that was the initial plan but then you changed you mind and decided to go for something more generic (with the venv backend). But it's fine to support this for conda envs at the beginning.